modules/yardstick/src/main/java/org/apache/ignite/yardstick/cache/load/IgniteCacheRandomOperationBenchmark.java

```diff
@@ -231,12 +279,47 @@ private void searchCache() throws Exception {
     }
 
     /**
+     * Load allowable operation from parameters.
+     */
+    private void loadAllowOperations() {
+        allowOperations = new ArrayList<>();
+        if (args.allowOperations().isEmpty())
+            Collections.addAll(allowOperations, Operation.values());
+        else
+            for (String opName : args.allowOperations())
```

Comment:
Please consider the following format "--allowedLoadTestOperation=PUT, PUT_ALL, GET".